tests/test_client_functional_oldstyle.py

```diff
@@ -868,7 +868,8 @@ def test_request_conn_closed(self):
     def test_keepalive(self):
         from aiohttp import connector
         with self.assertWarns(DeprecationWarning):
-            c = connector.TCPConnector(share_cookies=True, loop=self.loop)
+            c = connector.TCPConnector(share_cookies=True, loop=self.loop,
+                                       keepalive_timeout=30.0)
```

Comment:
Do you really need replacing default value with explicit timeout (equal to default though)?